### DISCUSSION TITLE
Discussions can now be closed.

### DISCUSSION THREAD

**Discussion by Elliria** [autokey/autokey#813] [Show and tell]
As of today, discussions can be closed. There should be a button with which to close this discussion below this.

**Comment by Elliria** [comment]
I see the button at this end. Do you see it, too, @josephj11, and can you use it, or can discussions only be closed by the people who opened them?

**Comment by josephj11** [comment]
I closed and reopened this discussion that you originated, but I probably have elevated permissions.

**Reply by Elliria** [reply]
Good. This is good news. You can now close any discussion and I can close any I started. That's as it should be.

**Comment by josephj11** [comment]
I just saw a post elsewhere that says is:opend and is:closedd don't work. Just use is:open and is":closed.

**Reply by Elliria** [reply]
Interesting. GitHub inserts the correct keywords in for us. I wonder why people are trying others instead.

**Comment by josephj11** [comment]
The announcement of the  feature specifed those predicates.
